src/utils/openvpn.ts

```diff
@@ -268,8 +269,19 @@ export class VpnManager extends EventEmitter {
 
 	public start(): Bluebird<true> {
 		this.process = spawn('/usr/sbin/openvpn', this.args(), {
-			stdio: 'ignore',
+			stdio: ['ignore', 'pipe', 'pipe'],
 		});
+		// proxy logs from the child process stdout/stderr
+		this.process.stdout.pipe(es.split()).pipe(
+			es.map((data: string) => {
+				this.emit('log', VpnLogLevels.n, data);
+			}),
+		);
+		this.process.stderr.pipe(es.split()).pipe(
+			es.map((data: string) => {
+				this.emit('log', VpnLogLevels.W, data);
+			}),
+		);
```

Comment:
Is this stream in flowing mode? We have to make sure it doesn't buffer forever

Reply:
If you're not going to do further processing it's better to do a `on('data', ..)` handler after you pipe to es.split. This will make the stream flowing and you don't have to use `es.map`.

However, this is not a good way of handling logs because there is no backpressure mechanism. If for whatever reason we get flooded with logs it will just accumulate in RAM.

Not for this PR, but logs should be normal streamed piped end-to-end without going through simple event emitters.